README.rst

```diff
@@ -467,27 +419,32 @@ NeMo LLM and Multimodal may need Megatron Core to be updated to a recent version
   cd megatron/core/datasets && \
   make
 
-
 NeMo Text Processing
 ~~~~~~~~~~~~~~~~~~~~
-NeMo Text Processing, specifically (Inverse) Text Normalization, is now a separate repository `https://github.com/NVIDIA/NeMo-text-processing <https://github.com/NVIDIA/NeMo-text-processing>`_.
 
-Docker containers
+NeMo Text Processing, specifically Inverse Text Normalization, is now a separate repository. It is located here: `https://github.com/NVIDIA/NeMo-text-processing <https://github.com/NVIDIA/NeMo-text-processing>`_.
+
+Docker Containers
 ~~~~~~~~~~~~~~~~~
-We release NeMo containers alongside NeMo releases. For example, NeMo ``r1.23.0`` comes with container ``nemo:24.01.speech``, you may find more details about released containers in `releases page <https://github.com/NVIDIA/NeMo/releases>`_.
 
-To use a pre-built container, please run
+NeMo containers are launched concurrently with NeMo version updates. For example, the release of NeMo ``r1.23.0`` comes with the container ``nemo:24.01.speech``. The latest containers are:
+
+* NeMo LLM and MM container - `nvcr.io/nvidia/nemo:24.03.framework`
```

Comment:
This should be updated to the unified container if its out already

Reply:
Submit a PR to update the container version. Stet as is.

Reply:
Updated to 24.05